Cpp_hands_on_programming/lib/pipe/Ipc_pipe.cc

```diff
@@ -0,0 +1,89 @@
+#include "Ipc_pipe.h"
+#include <iostream>
+#include <fcntl.h>
+#include <unistd.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <limits.h>
+
+void Ipc_pipe::send() {
+    std::cout << "Starting pipeSend.." << std::endl;
+	#if 1
```

Comment:
leftover from the times, method was not implemented?

Reply:
Sorry I am not sure what you mean here. Are you referring to #if #endif or something else?

Reply:
Yes, I think this '#if' should be removed, as function is implemented